### PR TITLE
deleted past calendar links, reformatted, added clarity to expectations

### PR DESCRIPTION
**Pull request by parispittman**

### CODE REVIEW

**Comment by BenTheElder** [comment]
ping CLA bot

File: mentoring/meet-our-contributors.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ When Slack seems like it’s going too fast, and you just need a quick answer fr
 Meet Our Contributors gives you a monthly one-hour opportunity to ask questions about our upstream community, watch interviews with our contributors, and participate in peer code reviews.
 
 ## When:
-Every first Wednesday of the month at the following times -
-* 03:30pm UTC [calendar link](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGw2OHM5bGRranJwZjI3OHN1MTk2Y2pjYjggcGFyaXNwaXR0bWFuQGdvb2dsZS5jb20&tmsrc=parispittman%40google.com)
-* 09:00pm UTC [calendar link](https://calendar.google.com/event?action=TEMPLATE&tmeid=N3M4dnZ0dGZwYTZzdXBuM3FndHRybGY0MTlfMjAxODAyMDdUMjEwMDAwWiBwYXJpc3BpdHRtYW5AZ29vZ2xlLmNvbQ&tmsrc=parispittman%40google.com&scp=ALL)
+Every first Wednesday of the month at the following times. Grab a copy of the calendar to yours from kubernetes.io/community
```
Comment (guineveresaenger) [inline mentoring/meet-our-contributors.md]: May we have a link? :)

**Comment by guineveresaenger** [comment]
/lgtm
Thanks @parispittman! <3

**Comment by guineveresaenger** [comment]
@parispittman I feel rude canceling your hold, that's why I'm not removing it...

**Pull request merged**